ibis/backends/tests/test_array.py

```diff
@@ -850,6 +850,15 @@ def test_zip(backend):
     assert len(s[0][0]) == len(res.type().value_type)
     assert len(x[0]) == len(s[0])
 
+    # clickhouse nested types can't be null
+    if backend.name() != "clickhouse":
```

Comment:
was lazy about using pytest connection fixtures, etc. I can split this up better if you want, but this isn't that ugly to me.